recipes/debugpy/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "debugpy" %}
+{% set version = "1.0.0b9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: d57e4854c78d007d9516b8d3ae0c827cd6df87374d6d56d9c4fbc7adfa83546b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  missing_dso_whitelist:
+    - /usr/lib/libc++.1.dylib  # [osx]
```

Comment:
This shouldn't be needed and points rather to a setup problem with the compilers. Can you remove it?